dev/com.ibm.ws.jpa.tests.container.client_fat/test-applications/BasicJPAMethodInjectClient.jar/src/com/ibm/ws/jpa/clientcontainer/fat/basic/methodinj/BasicJPAMethodInjAppClient.java

```diff
@@ -0,0 +1,296 @@
+package com.ibm.ws.jpa.clientcontainer.fat.basic.methodinj;
```

Comment:
Double check that all files have the appropriate IBM header. This is one of a couple I noticed without a header.

Reply:
THought I did, that's what reviews are for, thanks for the catch.